sdk/compute/Azure.ResourceManager.Compute/src/Customize/Models/CapacityReservationInstanceView.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.ResourceManager.Resources.Models;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Compute.Models
+{
+    public partial class CapacityReservationInstanceView
+    {
+        /// <summary> A list of all virtual machines resource ids allocated against the capacity reservation. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IReadOnlyList<SubResource> UtilizationInfoVirtualMachinesAllocated
+        {
+            get => UtilizationInfo?.VirtualMachinesAllocated;
```

Comment:
This seems problematic since before the list would have never been null and now if UtilizationInfo is null the list will be null.  Since its IReadOnlyList its probably safe to return an empty list here.

Reply:
Well I believe before this list also could be null
This is the code for this before this update:
```
        /// <summary> Unutilized capacity of the capacity reservation. </summary>
        internal CapacityReservationUtilization UtilizationInfo { get; }
        /// <summary> A list of all virtual machines resource ids allocated against the capacity reservation. </summary>
        public IReadOnlyList<SubResource> UtilizationInfoVirtualMachinesAllocated
        {
            get => UtilizationInfo?.VirtualMachinesAllocated;
        }
```
I just moved it here. `UtilizationInfo` could be null if the response does not contain it, and hence this list will be null if any one calls it

Reply:
This might be something that could updated on the safe flatten feature in the generator